### DISCUSSION TITLE
G7 7790 no Lights tab

### DISCUSSION THREAD

**Discussion by ivanprimestars** [T-Troll/alienfx-tools#472] [Q&A] [answered]
I installed the latest version of the utility (9.0.2) on my laptop, but the Lights tab is missing. I have read the manual, but found no answers. What should I do to make the Lights tab appear?
![Снимок экрана 2024-11-20 043415](https://github.com/user-attachments/assets/797a27db-ae57-4257-bcf1-3a6171a03345)
**UPD:**
I tried installing the utility using the zip archive and moved the files from DPTF to the Windows folder as written in the instructions, then I ran the **alienfx-cli probe** and got the message “Dell API not found, devices not found. Light devices not found, exiting!”
![Снимок экрана 2024-11-20 045114](https://github.com/user-attachments/assets/b068e3f5-fec1-4809-b027-88d0ec7e0976)
**UPD2:**
I just installed Alienware Command Center and now I get the message “Dell API ready, devices not found. Dell API version 5.2.0.0.0 detected”.

**Comment by T-Troll** [comment]
As CLI report you, no compatible light devices found into you system.
So, GUI disables lights functionality.

DPTF is for ESIF sensor naming from Intel DPTF, it's not about lights.

Are you sure you have Alienware keyboard at you system?

**Reply by ivanprimestars** [reply]
> Are you sure you have Alienware keyboard at you system?


Honestly I don't know, I have a G7 7790 version with white backlit keyboard (as far as I understand I can't change the color of the keyboard backlight in this version), I was just hoping I could turn off the light on the power button using your tool, but for some reason that doesn't work for me either. Maybe you have some ideas?
![IMG_1226](https://github.com/user-attachments/assets/e4414327-4a25-4e9f-b98e-08f8749a08c1)

**Reply by T-Troll** [reply, marked as answer]
It's just backlight keyboard, it can't be controlled.

**Reply by ivanprimestars** [reply]
Oh ok, anyway, thank you for your work.
